src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
```suggestion
* {
	box-sizing: border-box;
}

html {
  margin: 0;
  padding: 0;
  font-family: "Roboto", sans-serif;
}
```

Reply:
if i move margin and padding into html, it will work only for html tag, it means that body, ul will have defaul indents